protected/components/PatientSearch.php

```diff
@@ -204,6 +205,30 @@ public function getPatientName($term)
     }
 
     /**
+     * Fetch contact name and title from search term
+     */
+    public function getContactTitleName($term)
+    {
+        $contacts_result = Contact::model()->findAll(array('select'=>'title', 'distinct'=>true,));
+        $titles = array();
+        foreach ($contacts_result as $contact) {
+            array_push($titles, strtolower($contact['title']));
+        }
+
+        $terms = explode(' ', $term, 3);
+        $result = null;
```

Comment:
You could this line and the following line together
$result = array('contact_name' => array())